src/main/java/dumshenko/daniil/Square.java

```diff
@@ -0,0 +1,25 @@
+package dumshenko.daniil;
+
+public class Square extends Rectangle {
+
+    private double squareWidth;
+    private double squareHeight;
```

Comment:
nit: The idea here that you reuse the constructor from `Rectangle` and hence you don't need to declare the `height` and `width` in the `Square` class.
```suggestion
```